src/lib/evas/canvas/efl_canvas_textblock.eo

```diff
@@ -316,7 +316,7 @@ class Efl.Canvas.Textblock extends Efl.Canvas.Object implements Efl.Text,
              Default value is $[1].
              See @Efl.Text_Style.text_underline_height.
 
-           - $gfx_filter: Experimental filter name. See @Efl.Gfx.Filter for more information.
+           - gfx_filter: Experimental filter name. See Efl.Gfx.Filter for more information.
```

Comment:
What `$` and `@` do here? the first one is to see, `gtx_filter` as a variable a,d the second to set a link to `Efl.Gfx.Filter`?

Reply:
@Coquinho they're referencing other Eo classes. What I do here is prevent the reference, so the doc generator doesn't try to look for them.

Reply:
Oh, ok.
And why do we need to change docs here?

Reply:
Because otherwise it won't compile, since if the doc parser doesn't find the references, it gives a compilation error.